apps/explorer/src/app/components/txs/txs-per-block/index.tsx

```diff
@@ -24,28 +27,44 @@ export const TxsPerBlock = ({ blockHeight }: TxsPerBlockProps) => {
   });
 
   return (
-    <table>
-      <thead>
-        <tr>
-          <td>Transaction</td>
-          <td>From</td>
-          <td>Type</td>
-        </tr>
-      </thead>
-      <tbody>
-        {decodedBlockData &&
-          decodedBlockData.map(({ TxHash, PubKey, Type }, index) => {
-            return (
-              <tr key={index}>
-                <td>
-                  <Link to={`/txs/${TxHash}`}>{TxHash}</Link>
-                </td>
-                <td>{PubKey}</td>
-                <td>{Type}</td>
-              </tr>
-            );
-          })}
-      </tbody>
-    </table>
+    <div className="overflow-x-auto whitespace-nowrap mb-28">
+      <table className="w-full">
```

Comment:
chore (blocking): HTML here is invalid (divs used as children of this component and that is invalid. IMO this shouldn't be a table and just a header with a content body below it